packages/solid-repl/package.json

```diff
@@ -0,0 +1,67 @@
+{
+	"version": "0.24.0",
+	"name": "solid-repl",
+	"description": "Quickly discover what the solid compiler will generate from your JSX template",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/solidjs/solid-playground.git"
+	},
+	"files": [
+		"dist"
+	],
+	"homepage": "https://playground.solidjs.com",
+	"author": "Alexandre Mouton-Brady",
+	"module": "dist/repl.jsx",
+	"types": "src/types.d.ts",
+	"scripts": {
+		"dev": "vite",
+		"build": "tsc -p tsconfig.build.json && unocss \"./src/**\" && jiti build.ts",
+		"preview": "vite preview",
+		"format": "prettier -w .",
+		"tsc": "tsc"
+	},
+	"devDependencies": {
+		"@amoutonbrady/lz-string": "^0.1.0",
+		"@babel/core": "^7.22.9",
+		"@babel/plugin-syntax-jsx": "^7.22.5",
+		"@babel/preset-typescript": "^7.22.5",
+		"@babel/types": "^7.22.5",
+		"@solidjs/router": "^0.8.2",
+		"@types/babel__standalone": "^7.1.4",
+		"@types/dedent": "^0.7.0",
+		"@types/fs-extra": "^11.0.1",
+		"@unocss/cli": "^0.54.1",
+		"@unocss/preset-wind": "^0.54.1",
+		"acorn-jsx": "^5.3.2",
+		"assert": "^2.0.0",
+		"csstype": "^3.1.2",
+		"esbuild": "^0.18.17",
+		"fs-extra": "^11.1.1",
+		"jiti": "^1.19.1",
+		"jszip": "^3.10.1",
+		"mime": "^3.0.0",
+		"monaco-editor": "~0.40.0",
+		"prettier-plugin-tailwindcss": "^0.4.1",
+		"register-service-worker": "^1.7.2",
+		"typescript": "^5.1.6",
+		"unocss": "^0.54.1",
+		"vite": "^4.4.8",
```

Comment:
we shouldn't need a bunch of these packages for solid-repl (vite, lz-string, jszip, solid-router to name a few)